lens-driver-jdbc/src/main/java/org/apache/lens/driver/jdbc/JDBCDriver.java

```diff
@@ -556,6 +556,8 @@ public String getPlan() {
   public QueryCost estimate(AbstractQueryContext qctx) throws LensException {
     MethodMetricsContext validateGauge = MethodMetricsFactory.createMethodGauge(qctx.getDriverConf(this), true,
       VALIDATE_GAUGE);
+    String rewrittenQuery = rewriteQuery(qctx);
```

Comment:
Not sure why changes are done in estimate, when you are fixing bug in prepareQuery.

Reply:
In the azure-world, we are not going to have Lens interacting with the actual underlying drivers. https://docs.google.com/document/d/1K5Q3ZQZAEGkf472DZ_zuLbHZ9ycx86YAEhqbxO0o-aA/edit#


The cleaner way for me is to remove driverconnection which Lens is making for the explain part in 
https://github.com/apache/lens/blob/master/lens-driver-jdbc/src/main/java/org/apache/lens/driver/jdbc/JDBCDriver.java#L728